sound/soc/sof/intel/hda-ipc.c

```diff
@@ -75,6 +75,15 @@ void hda_dsp_ipc_get_reply(struct snd_sof_dev *sdev)
 	unsigned long flags;
 	int ret = 0;
 
+	/*
+	 * Sometimes, there is suspicious reply ipc arriving. The reply
+	 * ipc belongs to none of the ipcs sent from driver.
+	 * In this case, the driver must ignore the ipc.
+	 */
+	if (!msg) {
+		dev_info(sdev->dev, "suspicious ipc interrupt raised!\n");
```

Comment:
I think "spurious" or "unexpected" would be a better description for that interrupt - in the commit message, the comment and the print, in all patches. Also I'd make it a "dev_err" or at least a "dev_warn" because we want to investigate those casses.